homeassistant/components/zwave/config_flow.py

```diff
@@ -0,0 +1,85 @@
+"""Config flow to configure Z-Wave."""
+from collections import OrderedDict
+import logging
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import callback
+
+from .const import (
+    CONF_AUTOHEAL, CONF_POLLING_INTERVAL,
+    CONF_USB_STICK_PATH, CONF_NETWORK_KEY,
+    DEFAULT_CONF_AUTOHEAL, DEFAULT_CONF_USB_STICK_PATH,
+    DEFAULT_POLLING_INTERVAL, DOMAIN)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@callback
+def configured_hosts(hass):
```

Comment:
This is not needed. You are extending `ConfigFlow` and can just do `if self._async_current_entries():` to get current entries.